traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -199,6 +199,10 @@
             "title": "Global Max TPS",
             "desc": "The maximum transactions this delivery service can serve across all EDGE caches before traffic will be diverted to the bypass destination. For a DNS delivery service, the Bypass Ipv4 or Ipv6 will be used (depending on whether this was a A or AAAA request), and for HTTP delivery services the Bypass FQDN will be used."
           },
+	  "fqPacingRate": {
+	    "title": "Fair Queuing Pacing Rate Bps",
+            "desc": "The maximum bytes per second a cache will delivery on any single TCP connection. This uses the Linux kernel's Fair Queuing setsockopt(SO_MAX_PACING_RATE) to limit the rate of delivery. Traffic exceeding this speed will only be rate-limited and not diverted. This option requires 'net.core.default_qdisc = fq' in /etc/sysctl.conf"
```

Comment:
You might want to remove the

"This option requires 'net.core.default_qdisc = fq' in /etc/sysctl.conf"

This description is used in a tooltip in TP and the tooltip is intended for users of the system rather than administrators of the system. That comment is a little too technical for users in my opinion.

Reply:
removed that sentence